app/services/hyrax/flexible_schema_validators/class_validator.rb

```diff
@@ -79,7 +86,8 @@ def potential_existing_classes
         classes = @required_classes.dup
 
         Hyrax.config.registered_curation_concern_types.each do |concern_type|
-          classes << "#{concern_type.capitalize}Resource"
+          classes << "#{concern_type}Resource"
```

Comment:
Still wondering about this a bit... Does this mean that MobiusWorkResource would be considered valid as well as MobiusWork? Do we need to verify that the classes are actually valid, or do we do that somewhere else?